pom.xml

```diff
@@ -306,7 +306,7 @@
             <dependency>
               <groupId>com.puppycrawl.tools</groupId>
               <artifactId>checkstyle</artifactId>
-              <version>6.12.1</version>
+              <version>6.13</version>
```

Comment:
The problem with bumping plugin version quite often is that there are child modules (Malhar) outside Apex core repo and the plugin gets inherited to them. Any changes made in the configuration related to a particular version of checkstyle has then also be updated to outside child repositories.